### ISSUE TITLE
[Bug]: Window completely black on Linux Steam Deck

### ISSUE THREAD

**Issue by ChroniXmile** [Monitor221hz/Pandora-Behaviour-Engine-Plus#302]
### Check for duplicate issues

- [X] I have verified that there are no duplicate [issues](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/issues).

### Version

v2.3.0

### Provide Instructions on Reproducing the Problem

Launch program, get completely black window. The program is actually working, and after clicking around, I can run it, but cannot select any of the patches because I cant see anything.
I've attached the output I got when clicking around the black screen.

### Provide Relevant Files

[Output-Pandora-Testing.zip](https://github.com/user-attachments/files/16803254/Output-Pandora-Testing.zip)

### Provide Visual Proof (optional)

![pandora-black](https://github.com/user-attachments/assets/25c62c36-c681-4ea7-8827-832a5bff9f7f)

### Contributing To Development

- [X] I read this project's [Contributing To Development](https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/wiki/Contributing-To-Development)

**Comment by tovBlack** [comment]
I have a similar problem.
<img width="628" alt="image" src="https://github.com/user-attachments/assets/3bf7e7a8-48fb-47da-8c96-f4e7f4eacd42">

**Comment by tovBlack** [comment]
[Engine.log](https://github.com/user-attachments/files/16821443/Engine.log)

**Comment by zachboatwright** [comment]
using winetricks to download all dx3d components fixed this for me

**Comment by ChroniXmile** [comment]
> using winetricks to download all dx3d components fixed this for me

i tried this, but it didnt work. you do mean d3dx files right? which ones exactly?

**Comment by cataclym** [comment]
I also tried installing d3dx 9,10,11 - It is still blank.

**Comment by Mr-Benedict** [comment]
Same here. To be perfectly honest, I’m not 100% sure it’s the engine’s problem. 

If I run Pandora from protontricks it works, but it obviously can’t find anything because I’m not defining the “Start In” (if we could do that as an argument when launching Pandora standalone it would probably be a solution), if I run it from Vortex, it shows the black window.

@zachboatwright can you tell us how you launched it and what exactly have you installed to make it work for you?

**Comment by Phantatron** [comment]
I have exactly the same issue. Launching through mo2 running in GEproton 9.13 I get black screen, but if I launch mo2 through Bottles with "soda-9.0-1" runner, pandora works but mo2 freezes when closing it. Trying to figure out now if I can get it to work in that proton prefix, as mo2 and skyrim are working well there so I don't have to use this workaround to generate animations.

**Comment by Spaicrab** [comment]
After downloading d3decompiler_43 and d3decompiler_47 I stopped getting a black window! Although it looks broken (and very pink), it seems to be functional!

**Comment by ChroniXmile** [comment]
> After downloading d3decompiler_43 and d3decompiler_47 I stopped getting a black window! Although it looks broken (and very pink), it seems to be functional!

this worked for me

**Comment by ajdelguidice** [comment]
I don't know if you are using steamtinkerlaunch but it seems to cause a similar problem (as well as crashing) when you launch pandora from a mod manger that was launched through it (I've tried on both debian and arch with the same results). Launching the mod manager or pandora using protontricks seems to fix this.

**Comment by mthsgrc** [comment]
> After downloading d3decompiler_43 and d3decompiler_47 I stopped getting a black window! Although it looks broken (and very pink), it seems to be functional!

Thank you!

**Comment by Jongbloed83** [comment]
> After downloading d3decompiler_43 and d3decompiler_47 I stopped getting a black window! Although it looks broken (and very pink), it seems to be functional!

I tried but no success alas. Where did you put the dll's?

**Comment by ChroniXmile** [comment]
> I tried but no success alas. Where did you put the dll's?

i use protontricks to add the dll's to the wine prefix

**Comment by Jongbloed83** [comment]
> After downloading d3decompiler_43 and d3decompiler_47 I stopped getting a black window! Although it looks broken (and very pink), it seems to be functional!

I tried but no success alas. Where did you put the dll's?


> > I tried but no success alas. Where did you put the dll's?
> 
> i use protontricks to add the dll's to the wine prefix

I tried that, too. But protontricks doesn't show the vortex prefix to select, which I assume pandora uses. How did you get around this?

**Comment by EndlessFractal** [comment]
Using protontricks/winetricks and installing dxvk on the same prefix as the game makes it look somewhat better, very similar to #296...

![image](https://github.com/user-attachments/assets/1e9f1575-2bfd-42c6-bfdb-b990b70974b3)

**Comment by EndlessFractal** [comment]
Using protontricks/winetricks and installing dxvk using the Install windows components menu, I've managed to make it look like this:

![image](https://github.com/user-attachments/assets/a7aadeff-04de-49d4-82a6-f3fe5e989f65)

Not perfect, but very similar to #296 !

**Comment by EndlessFractal** [comment]
Using protontricks/winetricks and installing dxvk, I've managed to make it look like this:

![image](https://github.com/user-attachments/assets/a7aadeff-04de-49d4-82a6-f3fe5e989f65)

Not perfect, but very similar to #296 !

**Comment by Plemo0304** [comment]
I'd like to contribute a little solution I stumbled upon after trying for hours to get Pandora to work again(yes again because Pandora worked last weekend and after I upgraded my Skyrim finally to 1.6.1170 and setting up the main dependencies I got to Pandora and had to discover it shows a black screen on startup).
I've already tried the previously mentioned fixes(partly I already them like the d3d stuffs) and to the **solution I came to was use GE-Proton9-17 and higher**. I dunno which version below still work but GE-Proton9-5 does not.
Hope that helps someone.
